internal/kms/kms_ext_test.go

```diff
@@ -0,0 +1,127 @@
+package kms_test
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/boundary/internal/db"
+	"github.com/hashicorp/boundary/internal/iam"
+	"github.com/hashicorp/boundary/internal/kms"
+	"github.com/hashicorp/boundary/internal/types/scope"
+	"github.com/hashicorp/go-kms-wrapping/wrappers/aead"
+	"github.com/hashicorp/go-kms-wrapping/wrappers/multiwrapper"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// NOTE: This is a sequential test that relies on the actions that have come
+// before. Please see the comments for details.
+func TestKms(t *testing.T) {
+	t.Parallel()
+	ctx := context.Background()
+	conn, _ := db.TestSetup(t, "postgres")
+	rw := db.New(conn)
+	wrapper := db.TestWrapper(t)
```

Comment:
nit: I've taken to naming this `rootWrapper` in my newer tests just to be more explicit about its purpose.

Reply:
Probably it should be `extWrapper`. I actually ended up confusing myself a bit when writing the test because I was originally using `rootWrapper` as you mentioned but that wrapper is actually the external wrapper, not one of the internal keys.